src/EditorFeatures/Core/Implementation/InlineRename/IEditorInlineRenameService.cs

```diff
@@ -166,17 +181,10 @@ internal interface IInlineRenameInfo
         Glyph Glyph { get; }
 
         /// <summary>
-        /// The symbol kind of the symbol being renamed, for use in determining whether
-        /// the file can also be renamed
-        /// </summary>
-        SymbolKind SymbolKind { get; }
-
-        ImmutableArray<Location> OriginalDefinitionLocations { get; }
-
-        /// <summary>
-        /// The document that the inline rename was invoked from
+        /// Returns information about the file rename capabilities of 
+        /// an inline rename
         /// </summary>
-        DocumentId InvocationDocumentId { get; }
+        InlineRenameFileRenameInfo GetFileRenameInfo();
```

Comment:
Note: i'm pretty sure this interface is used by TS to support their rename scenario (i think i was the one that made the IEditorRenameXXX interfaces for this purpose.  So def check to see if what you're doing breaks them.

Reply:
Yup. TS is broken by this. Thanks for the heads up